interview-questions.md

```diff
@@ -8,33 +8,45 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does calling super do in a Ruby class?
 
-Your answer:
+Your answer: It is passed along with the invoked method.
 
-Researched answer:
+Researched answer: The INITIALIZE method INVOKE with an argument.
 
 2. What is a gem?
 
-Your answer:
+Your answer: 
 
-Researched answer:
+A gem is a package file for rails.
+
+Researched answer: 
+
+Its a library or snippet of code functionality. A little off.
 
 3. What is a relational database? Are there other kinds of databases?
 
-Your answer:
+Your answer: 
+
+Not a term Im really familiar with. Yes there is another type but its escapes me 
 
 Researched answer:
 
+ A type of database wit similar attributes that stores data and relate it to other databases
```

Comment:
Need some more research on what relational databases are as well as what other types of databases exist.